core/drivers/crypto/stm32/stm32_cryp.c

```diff
@@ -169,22 +169,26 @@
  */
 #define CLRBITS(v, bits)		((v) &= ~(bits))
 #define SETBITS(v, bits)		((v) |= (bits))
-#define CLRSETBITS(v, mask, bits)	((v) = ((v) & ~(mask)) | (bits))
+static inline void clrsetbits(uint32_t *v, uint32_t mask, uint32_t bits)
```

Comment:
remove `inline` attribute. useless.

Reply:
move below line 190, after macros, struct and variable definition

Reply:
> remove `inline` attribute. useless.

Agreed if we make sure the function is used below, otherwise the compiler will complain (-Wunused-function).